dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/QuayImageRegistry.java

```diff
@@ -110,6 +108,39 @@ public QuayImageRegistry(final HttpClient client, final ObjectMapper objectMappe
         return tags;
     }
 
+    public Tag updateTagWithImageInformation(Tool tool, Tag tag) {
```

Comment:
Seems more re-usable to just use a getImageInformation(Tool tool, String tagName) function instead